### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *resplendent-moxie-57dea2* failed.

### ISSUE THREAD

**Issue by JOBAYERAHAMED00** [JOBAYERAHAMED00/win11React#8]
### <span aria-hidden="true">❌</span> Deploy Preview for *resplendent-moxie-57dea2* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | be216bc2bc0dc0057c8ff0bbfde16c6e850d4d4b |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/resplendent-moxie-57dea2/deploys/67f0b44cd894f4000878edee |

_Originally posted by @JOBAYERAHAMED00 in https://github.com/JOBAYERAHAMED00/win11React/issues/7#issuecomment-2780207828_

**Comment by JOBAYERAHAMED00** [comment]
Issu
